addons/stock_picking_batch/models/stock_move_line.py

```diff
@@ -4,9 +4,9 @@
 from collections import defaultdict
 
 from odoo import _, fields, models
-from odoo import Command
+from odoo import Command, api
 from odoo.tools.float_utils import float_compare
-
+from odoo.osv import expression
 
```

Comment:
should follow the python imports [guideline](https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#python) 

Also, I believe it's PEP standard to leave 2 newlines before the class declaration